### ISSUE TITLE
No Lead

### ISSUE THREAD

**Issue by it300** [urbanyoung/Phasor#61]
Any chance at all that you are going to add no-lead mode?  People like it and i'm sure have been wishing for you to make it.

**Comment by urbanyoung** [comment]
Working on it now.

**Comment by ChadSki** [comment]
Is this issue closed because the feature is completed?

**Comment by urbanyoung** [comment]
Nope, I'm not too sure why he closed it. Maybe because he figured I'd actually do it? Well... hasn't quite happened yet...

**Comment by ChadSki** [comment]
Would you mind pushing your incomplete changes to a WIP branch? I'd love to take a stab at helping out.

**Comment by it300** [comment]
Yea I closed it because I knew it was being worked on.

**Comment by urbanyoung** [comment]
@ChadSki Whoops, sorry for the delay! I haven't done much on it but I'll polish it off over the weekend (well, get it at least working) and push a new crash.

**Comment by ChadSki** [comment]
Need any help?

**Comment by ChadSki** [comment]
@urbanyoung you wouldn't happen to have those unpushed changes would you? :smile:
